src/Integration/test/Integration.Test/Channel/QueueChannelTest.cs

```diff
@@ -13,338 +14,223 @@ namespace Steeltoe.Integration.Test.Channel;
 
 public sealed class QueueChannelTest
 {
+    private static readonly TimeSpan DefaultTestTimeout = TimeSpan.FromSeconds(5);
```

Comment:
@hananiel, please review the changes in this file

Reply:
A lot of these tests follow what's in spring closely - so deviating from it most likely will cause maintenance headaches. Second reading the diff, although the final pieces are same - I am not sure the tests are testing the same thing. The question is not how tasks work in C# but if the multithreaded context in which it will finally run is still valid - hence the synchronization mechanisms. I do not recommend this change.